<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="left_side_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="ClockWidget" name="left_top_widget" native="true"/>
      </item>
      <item>
       <widget class="DigitalTimeWidget" name="left_bottom_widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="right_side_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="right_top_widget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="CardStackWidget" name="stackedWidget"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="right_bottom_widget" native="true">
        <property name="styleSheet">
         <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(150, 150, 150, 100), stop:1 rgba(130, 130, 130, 255));
border-radius: 15px;
border: 2px solid rgba(0, 0, 0, 100);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClockWidget</class>
   <extends>QWidget</extends>
   <header>clockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DigitalTimeWidget</class>
   <extends>QWidget</extends>
   <header>digitaltimewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CardStackWidget</class>
   <extends>QStackedWidget</extends>
   <header>CardStackWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
